src/ooxml/testcases/org/apache/poi/xwpf/usermodel/TestColumn.java

```diff
@@ -0,0 +1,82 @@
+/* ====================================================================
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+==================================================================== */
+
+package org.apache.poi.xwpf.usermodel;
+
+import org.apache.poi.xwpf.XWPFTestDataSamples;
+import org.junit.Test;
+import java.io.IOException;
+import static org.junit.Assert.assertEquals;
+
+public class TestColumn {
+    @Test
+    public void testAddNewCol() throws IOException {
```

Comment:
Test name should be more descriptive.

```suggestion
    public void testAddNewColWithCorrectAmountOfColumns() throws IOException {
```